django_project/realtime/app_settings.py

```diff
@@ -215,6 +215,8 @@
     # terrain data
     '/home/headless/contexts/ash/context/idn_hillshade_wgs84.tif',
 ]
+VOLCANO_LAYER_PATH = (
+    '/home/headless/contexts/ash/context/idn_volcano_wgs84.shp')
```

Comment:
You have to find a way to override this from environment variables.
This is to include the file in the unittests. This file will not be found in Travis at the moment.

Reply:
So if we delete the old volcano and replace it with the new one and pointing the `VOLCANO_LAYER_PATH` there then we don't need to worry about the file will not be found on Travis right? Because it is the same pattern just like before CMIIW